src/module/helpers/struct_stress/Form.svelte

```diff
@@ -0,0 +1,79 @@
+<svelte:options accessors={true} />
+
+<script lang="ts">
+  import { createEventDispatcher } from "svelte";
+  import type { LancerActor } from "../../actor/lancer-actor";
+
+  export let title: string;
+  export let stat: "structure" | "stress";
+  export let lancerActor: LancerActor | null;
+
+  const dispatch = createEventDispatcher();
+
+  function focus(el: HTMLElement) {
+    el.focus();
+  }
+
+  function getIcon(kind: "structure" | "stress") {
+    if (kind === "stress") return "reactor";
+    return kind;
+  }
+
+  function getDamage(a: LancerActor) {
+    if (!a.is_mech() && !a.is_npc()) return 0;
+    return a.data.data.derived[stat].max - a.data.data.derived[stat].value + 1;
+  }
```

Comment:
This is fine, but the sveltier way to do it, given that these functions are only ever called with the current kind/actor, would be to use reactive variables.

```typescript
$: icon = stat === "stress" ? "reactor" : stat;
$: damage = getDamage(lancerActor);
```

```html
...
<i class="cci cci-{icon} i--m i--light" />
...
Roll {damage}d6 to determine what happens.
```